config_example.sh

```diff
@@ -1,287 +1,743 @@
 
-# Configure custom ntp servers if needed
-#export NTP_SERVERS="00.my.internal.ntp.server.com;01.other.ntp.server.com"
+################################################################################
+# CLUSTER_NAME - Openshift Cluster Name
+# Default: "ostest"
+#
+#export CLUSTER_NAME="mycluster"
+#
+################################################################################
 
-# Indicate number of workers to deploy
-#export NUM_WORKERS=0
+################################################################################
+# BASE_DOMAIN - Base Domain name for the OCP cluster
+# Default: "test.metalkube.org"
+#
+#export BASE_DOMAIN="kni.lab.metal3.io"
+#
+################################################################################
 
-# Indicate number of extra VMs to create but not deploy
-#export NUM_EXTRA_WORKERS=0
+################################################################################
+# IGNITION_EXTRA -
+# Provide additional master/worker ignition configuration, will be
+# merged with the installer provided config, can be used to modify
+# the default nic configuration etc
+#
+#export IGNITION_EXTRA=extra.ign
+#
+################################################################################
+
+################################################################################
+# ASSETS_EXTRA_FOLDER -
+# Folder where to copy extra manifests for the cluster deployment
+#
+#export ASSETS_EXTRA_FOLDER=local_file_path
+#
+################################################################################
 
-# Apply extra workers automatically if NUM_EXTRA_WORKERS is set
-#export APPLY_EXTRA_WORKERS=true
+################################################################################
+# FIPS_MODE
+# Enable FIPS mode
+#
+#export FIPS_MODE=true
+#
+################################################################################
 
-# Indicate the online status of the NUM_EXTRA_WORKERS set in extra_host_manifests.yaml
-#export EXTRA_WORKERS_ONLINE_STATUS=true
+################################################################################
+# DISABLE_MULTICAST -
+# In order to test using unicast for keepalived, one needs to disable multicast.
+# Setting this variable to true will block multicast via ebtables for both IPv4 and IPv6.
+#
+#export DISABLE_MULTICAST=false
+#
+################################################################################
+
+################################################################################
+# ENABLE_METALLB_MODE -
+# Enable metallb
+# Default is unset.
+# Choices: "l2", "bgp"
+#
+#export ENABLE_METALLB_MODE="l2"
+#
+# metallb container location (optional)
+#export METALLB_IMAGE_BASE=
+#export METALLB_IMAGE_TAG=
+#
+################################################################################
 
-# Provisioning interface on the helper ndoe
+####################################################################################################
+## Network Settings
```

Comment:
The subsections are a nice way to see the different types of configuration options.